### PR TITLE
[Big Tensor] Fix big tensor problem for paddle.nn.functional.class_center_sample

### PR DESCRIPTION
**Pull request by HeyDavid633**
主要修复：
1. label.numel() 为大Tensor时的错误抛出
2. label.numel() 导致 MemoryBuffer 分配尺寸溢出的错误抛出
3. 在 base_config.yaml 豁免了以上的错误抛出

原始的case尺寸，贴着 unint32 的上限设置；对于所有的case样例来说都是Torch error；即超过了 Torch 的限制
image

并在 tester 中增加了 case 以验证以上修复点2，尽管新增的case不是大Tensor

### CODE REVIEW

File: tester/api_config/8_big_tensor/big_tensor_error_0319_paddleonly.txt [hunk 1]
```diff
@@ -1742,6 +1742,9 @@ paddle.nn.functional.batch_norm(x=Tensor([715827883, 1, 2, 3],"float32"), runnin
 paddle.nn.functional.batch_norm(x=Tensor([715827883, 1, 3],"float64"), running_mean=Tensor([1],"float64"), running_var=Tensor([1],"float64"), weight=Tensor([1],"float64"), bias=Tensor([1],"float64"), )
 paddle.nn.functional.batch_norm(x=Tensor([715827883, 1, 3],"float64"), running_mean=Tensor([1],"float64"), running_var=Tensor([1],"float64"), weight=Tensor([1],"float64"), bias=Tensor([1],"float64"), epsilon=1e-05, )
 paddle.nn.functional.batch_norm(x=Tensor([715827883, 1, 3],"float64"), running_mean=Tensor([1],"float64"), running_var=Tensor([1],"float64"), weight=Tensor([1],"float64"), bias=Tensor([1],"float64"), epsilon=1e-05, data_format="NCL", )
+paddle.nn.functional.class_center_sample(Tensor([356493278],"int32"), 10, 8, )
+paddle.nn.functional.class_center_sample(Tensor([356493280],"int32"), 10, 8, )
+paddle.nn.functional.class_center_sample(Tensor([2294967295],"int32"), 10, 8, )
```
Comment (cangtianhuang) [inline tester/api_config/8_big_tensor/big_tensor_error_0319_paddleonly.txt:1747]: 新增配置放到 tester/api_config/8_big_tensor/big_tensor_1_8.txt

File: tester/base_config.yaml [hunk 2]
```diff
@@ -27,6 +27,9 @@ paddle_error_dismiss:
   paddle.linalg.solve: "(PreconditionNotMet) Input tensor 'A' or 'B' has too many elements (> INT_MAX)."
   paddle.nn.functional.group_norm: "(Unimplemented) GroupNorm kernel launch failed"
   paddle.einsum: "(Unimplemented) cublas GEMM does not support N >"
+  paddle.nn.functional.class_center_sample: 
+    - "(InvalidArgument) The total number of elements for 'label' should be less than"
+    - "(InvalidArgument) Illegal memory allocation, total allocated spacemust be greater than 0"
```
Comment (cangtianhuang) [inline tester/base_config.yaml:32]: 这个报错看上去是溢出了，不需要在paddle中修复嘛？

**Review by cangtianhuang — CHANGES_REQUESTED** [review]
torch error的case需要放到torch_error_skip中呀～

File: tester/base_config.yaml (hunk 2 above)
Comment (HeyDavid633) [reply tester/base_config.yaml:32]: Paddle 修复的方式在这里 https://github.com/PaddlePaddle/Paddle/pull/74364

**Review by cangtianhuang — APPROVED** [review]
LGTM
